cvFit/src/app/macroCalculator/calculator.service.ts

```diff
@@ -0,0 +1,96 @@
+import { Injectable } from "@angular/core";
+import { FormGroup } from "@angular/forms";
+
+@Injectable({
+    providedIn: 'root'
+})
+export class CalcService {
+    dailyCalories: number = undefined;
+
+    calculateBMR_male(macroForm: FormGroup){
+        enum maleBMRMultipliers{
+            inchConverter = 12,
+            heightMultiplier = 12.7,
+            weightMultiplier = 6.23,
+            ageMultiplier = 6.8,
+            additioinalValue = 66
+        }
+        return (((macroForm.get('heightF').value * maleBMRMultipliers.inchConverter) + macroForm.get('heightI').value) * maleBMRMultipliers.heightMultiplier) + (macroForm.get('weight').value * maleBMRMultipliers.weightMultiplier) - (macroForm.get('age').value * maleBMRMultipliers.ageMultiplier) + maleBMRMultipliers.additioinalValue
+    }
+    calculateBMR_female(macroForm: FormGroup){
+        enum femaleBMRMulitpliers{
+            inchConverter = 12,
+            heightMultiplier = 4.7,
+            weightMultiplier = 4.35,
+            ageMultiplier = 4.7,
+            additioinalValue = 655
+        }
+        return (((macroForm.get('heightF').value * femaleBMRMulitpliers.inchConverter) + macroForm.get('heightI').value) * femaleBMRMulitpliers.heightMultiplier) + (macroForm.get('weight').value * femaleBMRMulitpliers.weightMultiplier) - (macroForm.get('age').value * femaleBMRMulitpliers.ageMultiplier) + femaleBMRMulitpliers.additioinalValue
+    }
+    calculateDailyCalories(num: number, currentBMR: number){
+        enum exerciseMultipliers{
+            noExercise = 1.2,
+            moderateExercise = 1.375,
+            consistantExercise = 1.55,
+            intenseExercise = 1.725
+        }
+        if (num == 1){
+            return currentBMR * exerciseMultipliers.noExercise;
+        }
+        else if (num == 2){
+            return currentBMR * exerciseMultipliers.moderateExercise;
+        }
+        else if (num == 3){
+            return currentBMR * exerciseMultipliers.consistantExercise;
+        }
+        else if (num == 4){
+            return currentBMR * exerciseMultipliers.intenseExercise;
+        }
+        else 
+            console.log('Error with activity');
+            return -1;
+    }
+    goalFactor(num: number, calories: number){
+        enum goalMulitipliers{
+            calorieAdjustment = 500,
+            carbsPercentage = 0.4,
+            maintainCarbsPercentage = 0.5,
+            fatPercentage = 0.3,
+            protienPercentage = 0.3,
+            maintainProtienPercentage = 0.2,
+            carbDivisor = 4,
+            fatDivisor = 9,
+            protienDivisor = 4
+        }
+        if(num == 1){
```

Comment:
These are more of those magic numbers. No change for this one but you should try to make it clear to the code reader what is happening. Instead of 1 maybe it's an enum of `LoseWeight = 1`. Also consider renaming `num` to be something that describes what number it is intended to hold.